### DISCUSSION TITLE
Interpretation of Media Coefficients

### DISCUSSION THREAD

**Discussion by braggbear** [sibylhe/mmm_stan#13] [Q&A]
How can one interpret the media coefficients? For example, mdip_dm is centered around 0.07.  How can one interpret that 0.07?  Also, does that number factor in ad stock and diminishing returns?  Thank you!

**Comment by sibylhe** [comment]
Media coefficients indicate elasticity of the channel - how responsive y (sales) is to a change in x (media impression or spend). e.g., mdip_dm is 0.02, mdip_on is 0.11, mdip_on is more elastic, a unit change in mdip_on will result in more change in y. 

The media impression variables are adstocked, not hill-transformed, but the effect of impression on sales is diminishing in nature (as impression increases, the incremental sales it brings decreases), see [Cobb–Douglas function](https://en.wikipedia.org/wiki/Cobb%E2%80%93Douglas_production_function) and the visualization for detail.

**Reply by braggbear** [reply]
Thank you, Sibyl!  Is elasticity the same thing as cost per order (if your x is channel spend and y is orders)?

**Reply by sibylhe** [reply]
True in an additive MMM, but more precisely, it's $1 increase in ad spend will bring how many orders. The inverse of the coefficient is cost per order (CAC as you mentioned).

In multiplicative MMM, it's 1% change in x (spend) will lead to how many % change in y (sales). If a channel's coefficient is 0.02, 1% increase in spend will result in 0.02% increase in sales. 
Multiplicative MMM formula is: log y = B0 + B1 * log x1 + B2 * log x2. Small change in natural log of a variable is approximately equal to percentage change. See https://people.duke.edu/~rnau/411log.htm for details.

**Reply by robcurtis123** [reply]
Hello Sibyl. So should the CAC and ROAS not be affected but the Hill diminishing return function?  Is it separate?

Is the hill adjustment only used for the mROAS?

**Reply by sibylhe** [reply]
> So should the CAC and ROAS not be affected but the Hill diminishing return function? Is it separate?

True for overall ROAS of a given period (e.g., recent 1 year, 4 years).

> Is the hill adjustment only used for the mROAS?

Also for weekly ROAS.
